package.json

```diff
@@ -67,19 +67,19 @@
         "watch:labextension": "jupyter labextension watch ."
     },
     "dependencies": {
-        "@jupyterlab/application": "^4.4.5",
-        "@jupyterlab/apputils": "^4.5.5",
-        "@jupyterlab/cells": "^4.4.5",
-        "@jupyterlab/codeeditor": "^4.4.5",
-        "@jupyterlab/coreutils": "^6.4.5",
-        "@jupyterlab/docregistry": "^4.4.5",
-        "@jupyterlab/nbformat": "^4.4.5",
-        "@jupyterlab/notebook": "^4.4.5",
-        "@jupyterlab/rendermime": "^4.4.5",
-        "@jupyterlab/services": "^7.4.5",
-        "@jupyterlab/settingregistry": "^4.4.5",
-        "@jupyterlab/translation": "^4.4.5",
-        "@jupyterlab/ui-components": "4.4.5",
+        "@jupyterlab/application": "^4.4.6",
+        "@jupyterlab/apputils": "^4.5.6",
+        "@jupyterlab/cells": "^4.4.6",
+        "@jupyterlab/codeeditor": "^4.4.6",
+        "@jupyterlab/coreutils": "^6.4.6",
+        "@jupyterlab/docregistry": "^4.4.6",
+        "@jupyterlab/nbformat": "^4.4.6",
+        "@jupyterlab/notebook": "^4.4.6",
+        "@jupyterlab/rendermime": "^4.4.6",
+        "@jupyterlab/services": "^7.4.6",
+        "@jupyterlab/settingregistry": "^4.4.6",
+        "@jupyterlab/translation": "^4.4.6",
+        "@jupyterlab/ui-components": "^4.4.6",
```

Comment:
Oh, I guess it was failing because it was set to `4.4.5` without `^`.

Strictly JupyterLite 0.7.0a4 is based on Lab 4.5.0a3 so we may want to bump the minimum versions (and resolutions in yarn lock) to the pre-release packages to avoid package duplication at runtime.